closure-compiler-check/mediacapabilities.js

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2019 The Closure Compiler authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @fileoverview MediaCapabilities externs.
```

Comment:
May of these fields and types aren't used in this polyfill.  Maybe you should reduce to those needed to avoid maintaining this and this becoming out of date?

Reply:
> May of these fields and types aren't used in this polyfill. Maybe you should reduce to those needed to avoid maintaining this and this becoming out of date?

I defined the whole MC spec as it stood that day.  I did more than I needed for the polyfill so I could submit it directly to the Closure compiler team to include in a future release.